Resources/config/services.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+        http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service id="doctrine_bundle.migrations_collector"
+                 class="Doctrine\Bundle\MigrationsBundle\Collector\MigrationsCollector"
+                 public="true"
```

Comment:
no. Data collectors don't need to be public, so it is best to let the ContainerBuilder inline the service definition whenever possible by making it private.
Btw, we generally don't write ``public="true"`` as this is the default behavior.